### PR TITLE
changed gallery photo description to caption

### PR DESCRIPTION
**Pull request by rift137**

### CODE REVIEW

**Comment by rift137** [comment]
Pending ordering addition

File: spec/factories/gallery_photo_factory.rb [hunk 1]
```diff
@@ -5,6 +5,7 @@
     custom_profile
     article
     photo { File.open('spec/fixtures/upload.png') }
-    description "some description text"
+    caption "some caption text"
+    order 1
```
Comment (halloffame) [inline spec/factories/gallery_photo_factory.rb]: You should make this a sequence if you are validating uniqueness of order.

**Comment by halloffame** [comment]
:+1: looks good whenever tests pass

**Comment by rift137** [comment]
:+1:

**Pull request merged**